ext/curl/CMakeLists.txt

```diff
@@ -0,0 +1,12 @@
+#
+# Copyright (c) 2020 Nordic Semiconductor
+#
+# SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
+#
+
+zephyr_library()
```

Comment:
Generally about CMake files. I'm seeing a warning in compilation:

```
CMake Warning at /home/tor1/work/ncs/zephyr/CMakeLists.txt:710 (message):
  No SOURCES given to Zephyr library: ..__nrf__ext__curl

  Excluding target from build.

```

Reply:
Why is there a `zephyr_library()` here in the first place ?
None of the files: https://github.com/trantanen/sdk-nrf/blob/modem_shell/ext/curl/lib/CMakeLists.txt or https://github.com/trantanen/sdk-nrf/blob/modem_shell/ext/curl/tool/CMakeLists.txt uses `zephyr_library_sources()` so no files will ever be added to `zephyr_library()`

Reply:
Other responses should clarify this.